src/pages/drive/addIns/driveAddIns.tsx

```diff
@@ -0,0 +1,529 @@
+                    <td>None</td>
+                    <td>String</td>
+                </tr>
+                <tr>
+                    <td>state.driving</td>
+                    <td>Mobile device is detected as driving with the current vehicle</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.charging</td>
+                    <td>Mobile device is being powered</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.background</td>
+                    <td>Geotab Drive application is running in the background</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.online</td>
+                    <td>Mobile device has internet access</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.deviceCommunicating</td>
+                    <td>Telematics device is communicating to the server</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.gpsConnected</td>
+                    <td>Mobile device has GPS enabled</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>;
+
+
+const thirdPartyFromURISchema: ReactNode = <div className="paragraph">
+    <InformationalBox>
+        <p>Drive app v4.1.0+</p>
+    </InformationalBox>
+    <p>It's possible to open different applications like prontoforms or native calendar from add-ins. To do so, it's important to construct correct URI schema string and pass it to <code className="small-code-sample">window.open</code>. For example:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open(uriSchemaString, "_system")`}></CodeSample>
+    <p>Make sure to read carefully documentation of the app you're trying to open to use correct schema. For example, to open twitter application from addin you should use:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open("twitter://messages", "_system")`}></CodeSample>
+    <p>You can't use just <code className="small-code-sample">twitter://</code> as it's not correct and app won't open. You need to specify which page you want to open: <code className="small-code-sample">messages</code>, <code className="small-code-sample">account</code> etc.</p>
+    <p>To open webpage you need to use the same method, but with this notation:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open("https://google.com", "_blank")`}></CodeSample >
```

Comment:
```suggestion
        code={`window.open("https://google.com", "_blank")`}/>
```
This can be self closing cause it has no children inside of it.